migrate.go

```diff
@@ -24,13 +24,13 @@ type RedisMigrator struct {
 	migrationLogKey string
 }
 
-func NewRedisMigrator(client *redis.Client) *RedisMigrator {
+func NewRedisMigrator(client *redis.Client, prefix string) *RedisMigrator {
 	migrator := &RedisMigrator{
 		client:     client,
 		migrations: make([]*RedisMigration, 0),
 	}
 
-	migrator.SetKeyPrefix("redis-migrator:")
+	migrator.SetKeyPrefix(prefix)
```

Comment:
Thank you for the contribution!

I haven't yet decided if we should default to `"redis-migrator:` (or better `kvmigrator:` now that this library has a public name) if users call the constructor with an empty prefix. What do you think?

Depending on the outcome that should be included in the functions documentation.